package/home_assistant_intents/__init__.py

```diff
@@ -2,28 +2,24 @@
 import os
 import typing
 from pathlib import Path
-from typing import Any, Dict, Optional
+from typing import Any, Callable, Dict, IO, Optional
 
-try:
-    import importlib.resources
-
-    files = importlib.resources.files
-except (ImportError, AttributeError):
-    # Backport for Python < 3.9
-    import importlib_resources  # type: ignore
-
-    files = importlib_resources.files
+import importlib.resources
 
 _PACKAGE = "home_assistant_intents"
-_DIR = Path(typing.cast(os.PathLike, files(_PACKAGE)))
+_DIR = Path(typing.cast(os.PathLike, importlib.resources.files(_PACKAGE)))
```

Comment:
Why can't we do `Path(__file__).parent` ?

Reply:
It should allow the package to be zip safe, which I would need to mark in the `pyproject.toml`.